.github/workflows/ci_integrate.yml

```diff
@@ -25,11 +25,13 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-10.15, windows-2019]
-        python-version: [3.7, 3.9]
+        python-version: ['3.7', '3.10']
         requires: ['oldest', 'latest']
         exclude:
-          - {python-version: 3.7, requires: 'latest'}
-          - {python-version: 3.9, requires: 'oldest'}
+          - {python-version: '3.7', requires: 'latest'}
+          - {python-version: '3.9', requires: 'oldest'}
+          - {python-version: '3.10', requires: 'oldest'}
+          - {python-version: '3.10', os: 'windows-2019'}  # todo
```

Comment:
just curious: What's the issue here?

Reply:
I guess: https://discuss.pytorch.org/t/numpy-is-not-available-error/146192

Reply:
```suggestion
          - {python-version: '3.10', os: 'windows-2019'}  # todo: https://discuss.pytorch.org/t/numpy-is-not-available-error/146192
```